src/main/java/services/gorest/stepdefinition/UserStepsDefinition.java

```diff
@@ -0,0 +1,101 @@
+package services.gorest.stepdefinition;
+
+import cucumber.api.Scenario;
+import cucumber.api.java.After;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.restassured.response.Response;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import services.gorest.actions.user.CreateUser;
+import services.gorest.actions.user.DeleteUser;
+import services.gorest.actions.user.GetUser;
+import services.gorest.actions.user.UpdateUser;
+import services.gorest.models.responses.GetUserResponse;
+import services.gorest.validation.CommonValidations;
+import utils.methods.ReusableMethods;
+
+import static utils.variables.SessionVariableManager.getSessionVariable;
+import static utils.variables.SessionVariableManager.setSessionVariable;
+import static utils.variables.SessionVariables.*;
+
+@WithTags({
```

Comment:
Is there a reason why we are using tags here? This are used for tests so we can filter them when executing